Add Alkahraba, Alrajhi Bank, DEXTools, Malaa, SNB Mobile, Tameeni, Tawakkalna Services, Alinma App, ANB Telemoney, BingX, Bouki, Panda, Rabby Wallet, Tamara, Themer

Ahmed-Khalifa opened the pull request:
# Description
Alkahraba, Alrajhi Bank, DEXTools, Malaa, SNB Mobile, Tameeni, Tawakkalna Services, Alinma App, ANB Telemoney, BingX, Bouki, Panda, Rabby Wallet, Tamara, Themer

### Added
* Alkahraba (`sa.com.se.alkahraba/sa.com.se.alkahraba.MainActivity`)
* Alrajhi Bank (`com.alrajhiretailapp/com.alrajhiretailapp.MainActivity`)
* DEXTools (`io.dextools.app/io.dextools.app.ui.splash.SplashActivity`)
* Malaa (`tech.malaa.personal/tech.malaa.personal.MainActivity`)
* SNB Mobile (`com.snb.alahlimobile/com.snb.alahlimobile.MainActivity`)
* Tameeni (`tameeni.com/tameeni.com.MainActivity`)
* Tawakkalna Services (`sa.gov.nic.twkhayat/com.tawakkalna.twkapp.ui.SplashActivity`)
* Alinma App (`com.alinma.retail.mobile.v4/com.alinma.retail.mobile.v4.MainActivity`)
* ANB Telemoney (`telemoneyapp.anb.keystore/telemoneyapp.anb.keystore.MainActivity`)
* BingX (`pro.bingbon.app/pro.bingintl.ui.activity.MainActivity`)
* Bouki (`com.emcrey.riyadbank/com.emcrey.bouki.MainActivity`)
* Panda (`com.todoorstep.store/com.todoorstep.store.ui.activities.splash.SplashActivity`)
* Rabby Wallet (`com.debank.rabbymobile/com.debank.rabbymobile.MainActivity`)
* Tamara (`co.tamara.user/co.tamara.user.MainActivity`)
* Themer (`com.therxmv.telegramthemer/com.therxmv.telegramthemer.ui.MainActivity`)

### Updated
Alkahraba (`sa.com.se.alkahraba/sa.com.se.alkahraba.MainActivity`) After commiting changes

## Contributor's checklist
<!-- Replace [ ] with [x] to check. -->
- [x] I followed [the Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) (upd. Jan 2024) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.

Ahmed-Khalifa reviewed (commented):
I had a minor conflict and I resolved it in the last commit.
Kindly review.

Thank you.

x9136 commented on svgs/tawakkalna_services.svg:
This is a square icon: you need a different size, also take into account the limit on rounding.

Ahmed-Khalifa replied:
I changed the outer shape from squarcle to square with 8px corner radius.
![tawakkalna_services](https://github.com/LawnchairLauncher/lawnicons/assets/8474394/609e71a6-8ecb-4c6a-8cb3-2aec9605aa98)

Ahmed-Khalifa replied:
Changed the stroke to 12px

x9136 commented on app/assets/appfilter.xml:
[The names of the applications](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md#naming) are made incorrectly.

drawable="alinma_app" name="Alinma App" → drawable="alinma" name="الإنماء ~~ Anilma"
anilma_app.svg → anilma.svg

name="Alrajhi Bank" → name="مصرف الراجحي ~~ Alrajhi Bank"

name="Panda" → name="بنده ~~ Panda"

name="Tameeni" → name="تأميني ~~ Tameeni"

Ahmed-Khalifa replied:
Made the naming right, except (alinma app), I will leave it as is because the name in English is the correct one as it's shown in Play store, I only added the Arabic name to it.
I also added Arabic names to other applications:
name="Alinma App" → name="تطبيق الإنماء ~~ Alinma App"
name="Alkahraba" → name="الكهرباء ~~ Alkahraba"
name="Bouki" → name="بوكي ~~ Bouki"
name="SNB Mobile" → name="الأهلي موبايل ~~ SNB Mobile"
name="Tawakkalna Services" → name="توكلنا خدمات ~~ Tawakkalna Services"

x9136 replied:
> except (alinma app), I will leave it as is because the name in English is the correct one as it's shown in Play store

It's up to you, but I'll explain why I noted it. The "app" part doesn't make sense, because we're talking about the app anyway. If there were, for example, "Alinma app" and "Alinma game", then it would make sense. Also, the "app" in the name does not simplify the search in any way. In short, this part is useless, which means it's better to delete it. Otherwise, you decide for yourself.

x9136 commented on svgs/themer.svg:
![Screenshot 2024-05-01 at 10 28 54](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/4475a3a8-6629-46d0-9e1c-0ae3c8cdd4c1)
The brush stands out too much because of the blackness. It's worth redoing.

Ahmed-Khalifa replied:
I changed the brush stroke to 8px while the outer shape is 12px
![themer](https://github.com/LawnchairLauncher/lawnicons/assets/8474394/ca2ffe55-7a17-4ed1-ad69-35c54f4d37b7)

Ahmed-Khalifa replied:
I increased the paint brush stoke to 10px

x9136 commented on svgs/rabby_wallet.svg:
![Screenshot 2024-05-01 at 10 37 36](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/aa446620-cdc1-40e6-8987-43756f14f882)
It is worth simplifying the shape and slightly increasing the distances between the lines.

Ahmed-Khalifa replied:
I simplified the shape as suggested.
![rabby_wallet](https://github.com/LawnchairLauncher/lawnicons/assets/8474394/9ce7836e-de86-4a21-b5cb-c39de1af8ad5)

Ahmed-Khalifa commented on svgs/alinma_app.svg:
I set the stroke to 14px with a little bit of round conrners.
![alinma_app](https://github.com/LawnchairLauncher/lawnicons/assets/8474394/42a9bbdd-fe40-42e8-85ec-bec863759b6f)

Ahmed-Khalifa replied:
Fixed by centering the icon

Ahmed-Khalifa commented on svgs/alkahraba.svg:
I seperated the close strokes marked by aprox 20px, and the circle from fill to stoke.
![alkahraba](https://github.com/LawnchairLauncher/lawnicons/assets/8474394/c857644a-c524-4106-aba5-f7a5e4262670)

Ahmed-Khalifa commented on svgs/alrajhi_bank.svg:
I moved the icon towards the top edge of 160px box is it good or should I move it more towards the red area?
![alrajhi_bank](https://github.com/LawnchairLauncher/lawnicons/assets/8474394/2e8b2edb-c02e-41d5-a209-b0ae42aaa7a8)

Ahmed-Khalifa commented on svgs/anb_telemoney.svg:
I lowered the icon  to compensate the emptiness, tried to make the width between strokes the same (or close).
![anb_telemoney](https://github.com/LawnchairLauncher/lawnicons/assets/8474394/d2c3a07b-8197-4d78-a961-600ae14260b3)

x9136 replied:
> I lowered the icon to compensate the emptiness.

![Screenshot 2024-05-04 at 11 07 32](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/b0838afe-954a-4d6f-a304-f822969a1bc8)
Too much.



![Screenshot 2024-05-04 at 11 18 20](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/80c6edb5-3880-4bb0-b9f6-6e7ba0fd8004)
By the way. There are different options for how to make this icon.

Ahmed-Khalifa replied:
I couldn't determine the visual balance for this icon but I lowered the icon like aprox 10px.
The icons suggested are great but I will stick with the first option.
I hope you'll help me fix it if I don't meet the requirements you asked.

x9136 replied:
> I hope you'll help me fix it 

That's why I showed the picture with 3 variants of the current icon.

![Screenshot 2024-05-05 at 12 17 48](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/b0c3a77c-81d2-4b7d-848e-19536ad910f5)

Ahmed-Khalifa commented on svgs/bouki.svg:
moved the icon to the right edge of 160px box
![bouki](https://github.com/LawnchairLauncher/lawnicons/assets/8474394/18607bbb-6e10-4ccf-ac1d-55ffd617695d)

Ahmed-Khalifa commented on svgs/snb_mobile.svg:
Changed the stroke width to 14px
![snb_mobile](https://github.com/LawnchairLauncher/lawnicons/assets/8474394/471adb63-e1a7-46b3-99b7-4dc9be9fa4f8)

Ahmed-Khalifa commented on svgs/tameeni.svg:
I did some modifications in sizes, hope it will suffice.
![tameeni](https://github.com/LawnchairLauncher/lawnicons/assets/8474394/27febe50-cc17-4c0d-8013-b9d061350747)

Ahmed-Khalifa replied:
I reworked the icon again with correct height this time

Ahmed-Khalifa reviewed (commented):
I hope my last modifications are acceptable.

x9136 commented:
> I hope my last modifications are acceptable.

Could you add the icons in the PR instead of in the comments? Otherwise the same review has to be done twice.

Ahmed-Khalifa commented:
> > I hope my last modifications are acceptable.
> 
> Could you add updated icons in the PR instead of in the comments? Otherwise the same review has to be done twice.

I already commited the changes so the icons should be in the PR already.
I removed the app part from alinma_ app.
Deleted the icons from the comments.

Ahmed-Khalifa reviewed (commented):
Sorry I really took a lot from your time, but I'm really eager to contribute to this project so you'll have to bear with me.

Much appreciation and thanks.

x9136 commented:
> I hope you'll help me fix it 

That's why I showed the picture with 3 variants of the current icon.

![Screenshot 2024-05-05 at 12 17 48](https://github.com/LawnchairLauncher/lawnicons/assets/60105060/b0c3a77c-81d2-4b7d-848e-19536ad910f5)

Ahmed-Khalifa commented:
> > I hope you'll help me fix it
> 
> That's why I showed the picture with 3 variants of the current icon.
> 
> ![Screenshot 2024-05-05 at 12 17 48](https://private-user-images.githubusercontent.com/60105060/328006895-b0c3a77c-81d2-4b7d-848e-19536ad910f5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ5MjcwNzgsIm5iZiI6MTcxNDkyNjc3OCwicGF0aCI6Ii82MDEwNTA2MC8zMjgwMDY4OTUtYjBjM2E3N2MtODFkMi00YjdkLTg0OGUtMTk1MzZhZDkxMGY1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA1VDE2MzI1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY3YmM5YTMzMDlkM2M2OTc0Njk5Mzk5ODRmYTM1MzM4ZGFiMTRhODE5YWI2ZDA3N2UxMjM0NTBmZjE3ZDI3N2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QE3r8_Mahv2Rd0kXdYKFewnJj_wiHWS2R6yGV8w14Zg)

Fixed

Ahmed-Khalifa reviewed (commented):
Updated anb_telemoney
